app/main.py

```diff
@@ -2,0 +1,36 @@
+class Animal:
+    alive = []
```

Comment:
The `alive` list is currently a class attribute, which means it is shared across all instances of `Animal`. If the intention is to track living animals globally, this is fine. However, if each instance should have its own `alive` list, consider making it an instance attribute instead.